core/src/main/java/cz/xtf/core/openshift/HelmBinary.java

```diff
@@ -0,0 +1,54 @@
+package cz.xtf.core.openshift;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.Optional;
+
+import org.apache.commons.lang3.ArrayUtils;
+
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class HelmBinary {
```

Comment:
At the first look, this looks like duplication of:
https://github.com/xtf-cz/xtf/blob/master/core/src/main/java/cz/xtf/core/openshift/OpenShiftBinary.java

together with `CLIUtils` class above.

Could be `OpenShiftBinary` used instead? Or delegated?

Reply:
At least here is `helmConfigPath` which differentiate. Of course some code can be shared but conceptually I think it is good idea to have separate classes for OpenShiftBinary (`oc` command) and HelmBinary (`helm` command)

Reply:
Right, everything what is specific for `HelmBinary` can be in this class. This is about checking that code is not duplicated in `OpenShiftBinary`.